gen/src/generate/ucd/shared/version.rs

```diff
@@ -0,0 +1,57 @@
+use std::fs::File;
+use std::io::{self, Read, Write};
+use std::path::Path;
+use std::str::FromStr;
+
+use generate::PREAMBLE;
+
+use regex::Regex;
+
+pub struct UnicodeVersion(u16, u16, u16);
```

Comment:
We can put the type definition of `unic`'s `UnicodeVersion` in `unit-utils`, so we can use it here and not create a similar type with a different struct. What do you think?